tools/gatk_germlinecnvcaller_case.cwl

```diff
@@ -66,7 +66,7 @@ arguments:
       $(inputs.caller_external_admixing_rate == null ? "" : "--caller-external-admixing-rate " + inputs.caller_external_admixing_rate)
       $(inputs.disable_annealing == null ? "" : "--disable-annealing " + inputs.disable_annealing)
 
-      tar czf case-gcnv-tracking-shard-$(inputs.scatter_index).tar.gz -C out/case-tracking .
+      tar czf case-gcnv-tracking-shard-$(inputs.scatter_index).tar.gz -C out/case-tracking . || :
```

Comment:
tar has a transient failure where it will thinks a file has been modified during the tar process. I can't figure out why this happens and my investigation of the logs leads me to believe that it's not actually happening. Anyway this "error" is actually more of a warning. The tar process will continue without issue and deliver the complete tar. However, it does not actually exit 0 and Cavatica will call it a fail.

So what I've done here is to add an "or true" statement to these tar commands. Basically this will ignore any failure of these tar commands. It's a bit extreme and should not be used lightly. However, considering the transient nature of these errors and the fact that the errors are really just warnings with bad exit codes, I'm fine just assuming these tar commands are good.